index.html

```diff
@@ -73,44 +119,369 @@
     </div>
   </div>
 </div>
-<div class="row topics section">
-  <div class="col-xs-10 col-xs-offset-1 col-md-3 col-md-offset-2 col-sm-3 col-sm-offset-1">
-    <h3>Topics</h3>
+
+<div class="row beginner section">
+  <div class="col-xs-10 col-xs-offset-1 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 workshop">
+    <div class="row">
+      <div class="col-xs-12">
+        <a name="beginner"></a>
+        <h3>3 day Ember.js Jumpstart</h3>
+      </div>
+      <div class="col-xs-12">
+        <p>This 3 day on-location workshop <strong>takes participants through building a full Ember.js application</strong> using modern and idiomatic Ember.js patterns like Data down/Actions up, contextual components, closure actions etc. We'll <strong>introduce all of Ember.js' concepts</strong> required to create ambitious, production ready applications in a <strong>step by step manner</strong>.</p>
+        <p>While no previous Ember.js knowledge is required, participants <strong>should be familiar with JavaScript, jQuery, HTML, CSS and AJAX</strong> as well as the general concept of MVC.</p>
```

Comment:
Yeah, it's still MVC just the controller and view are combined in the component if you use components. If you use the classic template/controller approach then the template is the view basically.